src/nspawn/nspawn-seccomp.h

```diff
@@ -21,4 +21,23 @@
 
 #include <sys/types.h>
 
-int setup_seccomp(uint64_t cap_list_retain);
+#ifdef HAVE_SECCOMP
+#include "seccomp-util.h"
+
+typedef struct SyscallFilterSetCap {
+        SyscallFilterSet filter;
+        int capability;
+} SyscallFilterSetCap;
+#endif
```

Comment:
Is there a reason to have the typedef SyscallFilterSetCap inside the "#ifdef"?

I am not quite sure about systemd, but in the Linux kernel header files, they try to have the definitions being defined both in #ifdef block and the #else block, sometimes with dummy struct/functions definitions for the #else block.